lib/tsc.js

```diff
@@ -2352,7 +2352,6 @@ var ts;
         if (path.charCodeAt(1) === 58) {
             if (path.charCodeAt(2) === 47)
                 return 3;
-            return 2;
```

Comment:
please undo changes to files under `lib`

Reply:
done